config.go

```diff
@@ -28,47 +27,43 @@ type config struct {
 	button  *configButton
 }
 
-func prepareConfig() *config {
-
-	mqttServer, ok := os.LookupEnv("MQTT_SERVER")
-
-	if !ok {
-		mqttServer = "tcp://192.168.1.194:1883"
-	}
-
-	mqttTopic, ok := os.LookupEnv("MQTT_TOPIC")
+// Environment interface mimics the existing os package and the dependencies
+// we need for loading the config
+type Environment interface {
+	LookupEnv(key string) (string, bool)
+}
 
-	if !ok {
-		mqttTopic = "godor/door1"
-	}
+func prepareConfig() *config {
 
-	lePin, ok := os.LookupEnv("GODOR_PIN")
-	if !ok {
-		lePin = "1"
-	}
+	gpioPin := env("GODOR_PIN", "1")
```

Comment:
Lets rename this env var `GODOR_SENSOR_PIN`, so that way we can have a `GODOR_BUTTON_PIN` as well

Reply:
i thought about starting to tweak this but are we going to try to get 1 board running two sets of sensors/relays?

If so then the ENV vars dont make sense.... I was thinking if we go down that path to use a config.yml/json + viper package like the sensors.